src/nav/logs.yml

```diff
@@ -1,59 +1,34 @@
+  - title: Get started
+    path: /docs/logs/get-started
     pages:
-      - title: Get started
-        path: /docs/logs/log-management/get-started
-        pages:
-          - title: Get started with logs
-            path: /docs/logs/log-management/get-started/get-started-log-management
-          - title: Log management and security
-            path: /docs/logs/log-management/get-started/new-relics-log-management-security-privacy
-      - title: UI and data
-        path: /docs/logs/log-management/ui-data
-        pages:
-          - title: Logs UI
-            path: /docs/logs/log-management/ui-data/use-logs-ui
-          - title: Logs release notes
-            path: /docs/release-notes/logs-release-notes
-          - title: Log patterns
-            path: /docs/logs/log-management/ui-data/find-unusual-logs-log-patterns
-          - title: Query syntax for logs
-            path: /docs/logs/log-management/ui-data/query-syntax-logs
-          - title: Parsing
-            path: /docs/logs/log-management/ui-data/parsing
-          - title: Built-in parsing rules
-            path: /docs/logs/log-management/ui-data/built-log-parsing-rulesets
-          - title: Data partitions
-            path: /docs/logs/log-management/ui-data/data-partitions
-          - title: Drop filter rules
-            path: /docs/logs/log-management/ui-data/drop-data-drop-filter-rules
-          - title: Find data in long logs (blobs)
-            path: /docs/logs/log-management/ui-data/long-logs-blobs
-      - title: Log API
-        path: /docs/logs/log-management/log-api
-        pages:
-          - title: Introduction to the Log API
-            path: /docs/logs/log-management/log-api/introduction-log-api
-          - title: Log event data
-            path: /docs/logs/log-management/log-api/log-event-data
-          - title: TCP endpoint
-            path: /docs/logs/log-management/log-api/use-tcp-endpoint-forward-logs-new-relic
-      - title: Troubleshooting
-        path: /docs/logs/log-management/troubleshooting
-        pages:
-          - title: No data appears
-            path: /docs/logs/log-management/troubleshooting/no-log-data-appears-ui
-          - title: JSON message not parsed
-            path: /docs/logs/log-management/troubleshooting/json-message-not-parsed
-          - title: Live tail logging
-            path: /docs/logs/log-management/troubleshooting/view-log-messages-real-time-live-tail
-          - title: Log message is truncated
-            path: /docs/logs/log-management/troubleshooting/log-message-truncated
-          - title: Show surrounding logs
-            path: /docs/logs/log-management/troubleshooting/find-issues-cause-or-impact-surrounding-logs
+      - title: Get started with logs
+        path: /docs/logs/get-started/get-started-log-management
+      - title: Log management and security
+        path: /docs/logs/get-started/new-relics-log-management-security-privacy
+  - title: UI and data
```

Comment:
@barbnewrelic , I wonder if the UI node should be lower on the navigation, since users need to figure out how to forward their logs before they can see them in the UI. Here's an idea for the left-navigation:

* Get started
* Forward logs (also, maybe we need a document at the top of this node that reminds users that they all need to forward logs).
* Logs in context
* UI and data 
...

Reply:
Based on our slack conversation, I will move these around again. :-)